pkg/adapter/containers.go

```diff
@@ -786,3 +786,39 @@ func (r *LocalRuntime) Top(cli *cliconfig.TopValues) ([]string, error) {
 	}
 	return container.Top(descriptors)
 }
+
+// Port displays port information about existing containers
+func (r *LocalRuntime) Port(c *cliconfig.PortValues) ([]*Container, error) {
+	var (
+		portContainers []*Container
+		containers     []*libpod.Container
+		err            error
+	)
+	if !c.Latest && !c.All {
```

Comment:
We cut and paste this code all over the place, really need a common function for it.

Reply:
do you really want this done as part of a remote pr? if so, i can do i guess

Reply:
shortcuts.GetContainersByContext() with a filter parameter could cover this use-case.